kiva/tests/test_pdf_drawing.py

```diff
@@ -46,6 +46,7 @@ def draw_and_check(self):
         # been drawn.
         line = content.getData().splitlines()[-2]
         if not any((line.endswith(b'f'),
+                    line.endswith(b'Q'),
```

Comment:
not sure what this special incantation is.

Reply:
This is what it in the PDF when you draw an image.